TestPlaybooks/playbook-ReversingLabsTitaniumCloud-Test.yml

```diff
@@ -0,0 +1,216 @@
+id: 01f92c20-349f-4ced-877e-30476d014fe3
+version: 2
+name: ReversingLabsTitaniumCloud-Test
```

Comment:
Currently the playbook has no inputs. Incidents needs to have some hashes in order this test playbook enrich the hashes. You need to insert some hard coded hashes into the context/incident and then validate if the enrich tasks really retrieved the correct data. 
The playbook need to be able to run automatically without anyone run it manually or put him some data

Reply:
I'll add manual input